content/en/docs/ops/best-practices/security/index.md

```diff
@@ -7,36 +7,170 @@ owner: istio/wg-security-maintainers
 test: no
 ---
 
-This section provides some deployment guidelines to help keep a service mesh secure.
-
-## Use namespaces for isolation
```

Comment:
I wonder if there should be an anti-patterns section (things not to do) and one of the items should be not relying too much on namespaces for isolation?